workflow-templates/check-shell-task.yml

```diff
@@ -93,7 +93,7 @@ jobs:
 
       - name: Download latest ShellCheck release binary package
         id: download
-        uses: MrOctopus/download-asset-action@1.0
+        uses: MrOctopus/download-asset-action@1.1
```

Comment:
An alternative approach to installation would be **APT**, which is an officially supported installation method:

https://github.com/koalaman/shellcheck#installing

However, the version in the Ubuntu 22.04 package repository is outdated:

https://launchpad.net/ubuntu/jammy/+source/shellcheck

So I think that is an unsuitable method.

---

The Snap package is another officially supported installation method, and the package is up to date:

https://snapcraft.io/shellcheck

So I think this would be the best alternative to the current release binary download installation method.